permissions/plugin-commons-collections4-api.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "commons-collections4-api"
+github: &GH "jenkinsci/commons-collections4-api-plugin"
+paths:
+- "io/jenkins/plugins/commons-collections4-api"
+developers:
+- "warden"
+issues:
+  - github: *GH
```

Comment:
isn't that missing security section?
https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-jira.yml#L12